torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1138,14 +1138,34 @@ class IrParser {
                 c10::nullopt, value_map[node->inputs()[0]->unique()]);
             auto operand = list_val.front();
             list_val.pop_front();
-            Val* low = value_map.count(node->inputs()[1]->unique()) != 0
+            bool has_low = value_map.count(node->inputs()[1]->unique()) != 0;
+            Val* low = has_low
                 ? *value_map[node->inputs()[1]->unique()]
                 : IrBuilder::create<Double>(std::numeric_limits<float>::min());
-            Val* high = value_map.count(node->inputs()[2]->unique()) != 0
+            bool has_high = value_map.count(node->inputs()[2]->unique()) != 0;
+            Val* high = has_high
                 ? *value_map[node->inputs()[2]->unique()]
                 : IrBuilder::create<Double>(std::numeric_limits<float>::max());
```

Comment:
You still have these very misleading `high` and `low` assignments, you should be able to remove them?